src/utils/secureDb.ts

```
@@ -0,0 +1,367 @@
+        return { isValid: false, errors };
+      }
+      if (input.length > (opts.maxLength || 1000)) {
+        errors.push(`String too long (max ${opts.maxLength || 1000} characters)`);
+      }
+      if (!opts.pattern.test(input)) {
+        errors.push('String contains invalid characters');
+      }
+      break;
+
+    case 'number':
+      const numValue = typeof input === 'string' ? parseInt(input, 10) : input;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)